plugins/backend/backstage-plugin-argo-cd-backend/src/service/argocd.service.ts

```diff
@@ -83,15 +85,12 @@ export class ArgoService implements ArgoServiceApi {
   async getRevisionData(
     baseUrl: string,
     options: {
-      name?: string;
-      selector?: string;
```

Comment:
I guess this `selector` was never actually filled in on any calling site?

Reply:
yes.